SKPhotoBrowser/SKPhotoBrowserDelegate.swift

```diff
@@ -63,15 +63,25 @@ import Foundation
     @objc optional func removePhoto(_ browser: SKPhotoBrowser, index: Int, reload: @escaping (() -> Void))
     
     /**
-     Asks the delegate for the view for a certain photo. Needed to detemine the animation when presenting/closing the browser.
+     Asks the delegate for the view for a certain photo. Needed to detemine the animation when presenting the browser.
      
      - Parameter browser: reference to the calling SKPhotoBrowser
      - Parameter index: the index of the removed photo
      
      - Returns: the view to animate to
      */
-    @objc optional func viewForPhoto(_ browser: SKPhotoBrowser, index: Int) -> UIView?
+    @objc optional func presentationViewForPhoto(_ browser: SKPhotoBrowser, index: Int) -> UIView?
```

Comment:
why do we want to separate the delegate calls? so we can scroll to the image properly in ad detail?

Reply:
With just one method you are suppling just one view for the present and dismiss animation. We don't want the dismiss animation as it does not fit very well with our image slider. 
So, by splitting them we can use the presentation animation alone.